direct/data/transforms.py

```diff
@@ -153,7 +186,20 @@ def fft2(
     return data
 
 
-def ifft2(
+def fft2(
+    data: torch.Tensor,
+    dim: Tuple[int, ...] = (1, 2),
+    centered: bool = True,
+    normalized: bool = True,
+) -> torch.Tensor:
+
+    if data.requires_grad:
+        return origin_fft2(data, dim, centered, normalized)
+    else:
+        return FFTONNX.apply(data, dim, centered, normalized)
```

Comment:
Please provide an example of a reconstructed image with our `fft2`/`ifft2` torch function compared to a reconstructed image with `FFTONNX`.  

What does `torch.allclose(origin_fft2(data, dim, centered, normalized), FFTONNX.apply(data, dim, centered, normalized))` returns?

Reply:
Which demo can be used to provide an example of a reconstructed image?

Can I use a random `data` for comparison?

Reply:
```python
import torch
from direct.data import transforms as T
from tests.tests_nn.test_rim import create_input

def main():
    inp = create_input([1, 3, 64, 64, 2])

    ref = T.origin_fft2(inp, (1, 2), True, True)
    out = T.FFTONNX.apply(inp, (1, 2), True, True)

    print(torch.allclose(ref, out)) # True

if __name__ == "__main__":
    main()
```

I use the script above for comparison and `torch.allclose(ref, out)` returns `True`.

Reply:
Done